portal/models/scheduled_job.py

```diff
@@ -80,13 +82,28 @@ def crontab_schedule(self):
                        day_of_week=svals[4]
                       )
 
+    def trigger(self):
+        from .. import tasks
+        func = getattr(tasks, self.task, None)
+        if func:
+            args_in = self.args.split(',') if self.args else []
+            kwargs_in = self.kwargs or {}
+            try:
+                msg = func(*args_in, job_id=self.id, **kwargs_in)
```

Comment:
Granted, you're naming the `job_id` param, but shouldn't the order be `(job_id, *args, **kwargs)` ?  that's what i'm more familiar with seeing and i think it'll possibly bind incorrectly otherwise.

Reply:
Mmm, I think it'll work this way...at least, it worked for my testing (albeit with empty args/kwargs), and that's what I've been doing in `celery_worker.py`. But I can test in the suggested order and see if that works.

Reply:
Oh, nevermind, you're right. Changing now (and adding additional tests).